daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/DFDLDefineVariable.scala

```diff
@@ -177,7 +176,7 @@ final class DFDLSetVariable(node: Node, decl: AnnotatedSchemaComponent)
   extends VariableReference(node, decl)
   with DFDLSetVariableRuntime1Mixin {
   private lazy val attrValue = getAttributeOption("value")
-  private lazy val <dfdl:setVariable>{eltChildren @ _*}</dfdl:setVariable> = node
+  private lazy val eltChildren = node.nonEmptyChildren
```

Comment:
This is ok, but it is not equivalent as it does not verify that the node is a setVariable element. 
This would be better:
```
private lazy val Elem("dfdl", "setVariable", _, _, eltChildren) = node
```